pkg/i2gw/providers/gce/gce.go

```diff
@@ -32,16 +34,25 @@ func init() {
 
 // Provider implements the i2gw.Provider interface.
 type Provider struct {
-	storage   *storage
-	reader    reader
-	converter converter
+	storage                       *storage
+	reader                        reader
+	resourcesToIRConverter        resourcesToIRConverter
+	irToGatewayResourcesConverter irToGatewayResourcesConverter
 }
 
 func NewProvider(conf *i2gw.ProviderConf) i2gw.Provider {
+	// Add BackendConfig and FrontendConfig to Schema when reading in-cluster
```

Comment:
Nit: Comment includes "FrontendConfig" but it looks like you're only adding "BackendConfig" below.

Reply:
Thanks for catching this, updating the comment.